resources/views/community/components/user/profile/last-seen-in.blade.php

```diff
@@ -0,0 +1,62 @@
+@props([
+    'userMassData' => [],
+])
+
+<?php
+use App\Platform\Models\PlayerSession;
+use App\Platform\Models\Game;
+use Illuminate\Support\Carbon;
+
+$mostRecentSession = PlayerSession::where('user_id', $userMassData['ID'])
+    ->with('game')
+    ->orderBy('created_at', 'desc')
+    ->first();
+
+// If there's no session for some reason, try to fall back to the user record.
+$sessionGame = $mostRecentSession?->game;
+if (!$mostRecentSession) {
+    $sessionGame = Game::with('system')->find($userMassData['LastGame']['ID']);
+}
+
+$mostRecentRichPresenceMessage = (
+    $mostRecentSession?->rich_presence
+    ?? $userMassData['RichPresenceMsg']
+    ?? null
+);
+?>
+
+@if ($sessionGame)
+    <div class="mb-6">
+        <div class="flex w-full items-center gap-x-1.5">
+            <p role="heading" aria-level="2" class="mb-0.5 text-2xs font-bold">
+                Most recently played
```

Comment:
the timestamp is slightly misaligned with this text
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/1d1cb372-c0de-4282-8079-91678e0eb96d)

Reply:
Good catch, fixed in latest.